### DISCUSSION TITLE
Bootcamp Windows Startup Issues

### DISCUSSION THREAD

**Discussion by JedNZ** [dakanji/RefindPlus#118] [General]
I'm running MyBootMgr 0.8.2a on my cMP 4.1>5.1. OC and OC_ALT are both working fine booting to macOS 11.6.8. I can also boot natively to Mojave from the RefindPlus screen, however I can't boot to my Windows 11 SSD - instead it gives me a screen full of text and then stalls before the Windows logo is displayed. 

The Windows 11 SSD boots fine in Parallels, so I know it does run. 

The text that scrolls on the screen includes multiple entries starting with "efi_fusion_pairing..." and "APFSStart ... Mounting with apfs_efi_osx-..."

I can see there is an extra x64_ExFatDxeLegacy.efi file now in the BOOT > drivers folder - do I need to remove that maybe?

I had no issues booting the Windows 11 SSD (which is in SATA bay 1) on my previous version of MyBootMgr v0.7.9b, so I'm guessing something in the RefindPlus EFI>BOOT folder has changed such that it's causing interference with Windows booting.

I have compared the config.conf files of both versions with FileMerge but can't identify from the changes what might be causing the issue. 

I appreciate any help to problem solve this.

**Comment by dakanji** [comment]
Please raise an actual issue.
You can do this by choosing the `Bug Report` option here: https://github.com/dakanji/RefindPlus/issues/new/choose

**Reply by JedNZ** [reply]
Sorry about that. New to GitHub
